src/DDTrace/Integrations/DatabaseIntegrationHelper.php

```diff
@@ -44,9 +44,18 @@ public static function injectDatabaseIntegrationData(HookData $hook, $backend, $
             }
 
             $span = $hook->span();
-            $databaseService = $span->meta['peer.service'] ?? $span->service;
-
-            $query = self::propagateViaSqlComments($hook->args[$argNum], $databaseService, $propagationMode);
+            $targetHost = $span->meta[Tag::TARGET_HOST] ?? '';
+            $dbName = $span->meta[Tag::DB_NAME] ?? $span->meta[Tag::DB_INSTANCE] ?? '';
+            $peerService = $span->meta['peer.service'] ?? '';
```

Comment:
nit: this seems like the only tag not specified as `Tag::...` and I'm not sure if that's intentional or not

Reply:
Yes, this was sort of intentional since there is no `Tag::PEER_SERVICE`. I can add it - won't make any harm